### ISSUE TITLE
useAuthState initialising not working

### ISSUE THREAD

**Issue by geraldoramos** [CSFrequency/react-firebase-hooks#1]
Hi,

I've been trying the `useAuthState` hook and looks like the `initialising` variable is not working.

Here is the code:
```javascript

export default function Login() {

  const { initialising, user } = useAuthState(firebaseAuth());
  const login = () => {
      firebaseAuth().signInWithEmailAndPassword('teste@test.com', 'test123');
    };
  const logout = () => { firebaseAuth().signOut(); };
  
    if (initialising) {
      return (
        <div>
          <Spinner marginX="auto" marginY={50} />
        </div>
      )
    }
    if (user) {
      return (
        <div>
          <p>Current User: {user.email}</p>
          <Button onClick={logout}>Log out</Button>
        </div>
      );
    }
    return (
      <Button onClick={login}>Log in</Button>
    )
  }
```

The login/logout works fine, but I've confirmed that the `initialising` block is never reached.

Thanks in advance,

Geraldo

**Comment by chrisbianca** [comment]
Thanks for flagging - this was an error on my part when refactoring to use the underlying `useReducer` hook rather than `useState`.

I've just published v0.3.2 which fixes the problem.

Please let me know how you get on in general and whether you have any feedback on how well they work, what's missing, that sort of thing...

**Comment by geraldoramos** [comment]
Sorry for the delay. Thanks a lot, it's working great for me.
